packages/@okta/vuepress-theme-prose/components/FrontPageCodeMirror.vue

```diff
@@ -35,7 +35,7 @@ export default {
 
 var widget = new OktaSignIn({
   baseUrl: "{{yourOktaDomain}}",
-  logo: "/sites/all/themes/developer/owb/alliance.png",
+  logo: "/alliance.png",
```

Comment:
@albertchen-okta 
![image](https://user-images.githubusercontent.com/72248639/107274565-923d8300-6a1e-11eb-915e-74544f5c4a35.png)

Reply:
the codemirror code isn't actually used to render the widget at all right now, so this URL was never real anyways. however updating it definitely makes it cleaner.